src/functionalTest/resources/fixtures/maintenance/patch/patch.json

```diff
@@ -1,6 +1,4 @@
 {
-  "eventData" : {
-    "divorceOrDissolution": "dissolution",
-    "id": "caseId"
-  }
+  "divorceOrDissolution": "dissolution",
```

Comment:
did u run the functional test with this input json ? 
Request body is a map for patch case ?

Reply:
The reason I removed eventData, was because this was not being used and was just stripped before actual case patch data was sent onto CMS.  I tested this with the functional tests and it all works.